### ISSUE TITLE
Fatal Error: undefined method `occurrenceDateTime'

### ISSUE THREAD

**Issue by darena-mhaque** [onc-healthit/inferno#263]
I am using the online tool for R4.

I am receiving the error `Fatal Error: undefined method 'occurrenceDateTime'.........`. It's a pretty long error. This is only happening for the Procedure request for test numbers 3 and 4. Here is a screenshot of the error:

![image](https://user-images.githubusercontent.com/45796503/61961975-869e7e80-af96-11e9-832e-8084d0135e21.png)

The result there says 0 requests were made, so I'm wondering if this error occurred before the request was even made.

I'll copy paste the error below in text for easier access:

```
Error Message: Fatal Error: undefined method `occurrenceDateTime' for #<FHIR::Procedure:0x00007fa6ced57678 @id="5d3a089b72e89ba0945a1ef6", @meta=#<FHIR::Meta:0x00007fa6ced56688 @id=nil, @extension=[], @versionId="1", @lastUpdated="2019-07-25T19:53:02.557+00:00", @source=nil, @profile=[], @security=[], @tag=[]>, @implicitRules=nil, @language=nil, @text=nil, @contained=[], @extension=[], @modifierExtension=[], @identifier=[#<FHIR::Identifier:0x00007fa6ced55be8 @id=nil, @extension=[], @use=nil, @type=nil, @system="http://terminology.bluebuttonpro.com/identifier", @value="6e111e681224433592ac6baa5a451b68_5d3a089b72e89ba0945a1ef6", @period=nil, @assigner=nil>], @instantiatesCanonical=[], @instantiatesUri=[], @basedOn=[], @partOf=[], @status="entered-in-error", @statusReason=nil, @category=nil, @code=#<FHIR::CodeableConcept:0x00007fa6ced555f8 @id=nil, @extension=[], @coding=[#<FHIR::Coding:0x00007fa6ced550f8 @id=nil, @extension=[], @system="http://snomed.info/sct", @version=nil, @code="1698001", @display="Ulcer of bile duct", @userSelected=nil>], @text=nil>, @subject=#<FHIR::Reference:0x00007fa6ced54a40 @id=nil, @extension=[], @reference="Patient/5d3a089b72e89ba0945a1ea9", @type=nil, @identifier=nil, @display=nil>, @encounter=nil, @performedDateTime="2017-09-10T08:15:49.0000000Z", @performedPeriod=nil, @performedString=nil, @performedAge=nil, @performedRange=nil, @recorder=nil, @asserter=nil, @performer=[], @location=nil, @reasonCode=[], @reasonReference=[], @bodySite=[], @outcome=nil, @report=[], @complication=[], @complicationDetail=[], @followUp=[], @note=[], @focalDevice=[], @usedReference=[], @usedCode=[]> 
```

**Comment by czh-orz** [comment]
Hi, 
Thanks for bringing this to our attention. This is happening because the metadata for the date search parameter for procedures is incorrect. There is no occurenceDateTime in the Procedure resource. http://hl7.org/fhir/us/core/SearchParameter-us-core-procedure-date.html

We'll look into resolving this soon.

Thanks, 
Chase

**Comment by Jammjammjamm** [comment]
This was addressed by #364, but has not yet been deployed.

**Comment by yunwwang** [comment]
Fix is released in v2.9.0
